sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SearchFieldBase.java

```diff
@@ -46,4 +46,6 @@ public String getName() {
     public SearchFieldDataType getDataType() {
         return dataType;
     }
+
+    public abstract SearchField toSearchField();
```

Comment:
Im not understanding the purpose of this. It's also missing Javadocs if we plan on keeping it.